consensus/state.go

```diff
@@ -810,7 +810,7 @@ func (cs *State) handleMsg(mi msgInfo) {
 		// We probably don't want to stop the peer here. The vote does not
 		// necessarily comes from a malicious peer but can be just broadcasted by
 		// a typical peer.
-		// https://github.com/lazyledger/lazyledger-core/issues/1281
+		// https://github.com/tendermint/tendermint/issues/1281
```

Comment:
Is there a reason for this change? Ref: https://github.com/lazyledger/lazyledger-core/pull/152#discussion_r565445956

Reply:
Yeah, this change was not intentional. There is no issue with this number in ll-core, only in the tendermint repo. Probably, a search and replace hickup while going through merge-conflicts when we've updated from upstream.

Reply:
Ah, so it's fixing a previous bad-replace. Got it.